tests/tests.py

```diff
@@ -676,11 +676,11 @@ def test_remote_gs():
 @pytest.mark.skip(reason="Need to choose how to provide billable project")
 @connected
 @not_ci
-def test_pr79(
+def test_issue96(
```

Comment:
ah one tiny tweak! The previous test_issueX I believe are referencing BitBucket (the two issue trackers don't line up) so here we would want a test_github_issueX instead. @johanneskoester is this correct?

Reply:
haha I was thinking about asking what happens once the github issues catch up -- I'll go ahead an push an update with the stated naming convention.